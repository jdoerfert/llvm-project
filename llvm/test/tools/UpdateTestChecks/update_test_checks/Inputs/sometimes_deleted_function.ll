; RUN: opt -S < %s | FileCheck %s --check-prefixes=ALL,FIRST
; RUN: opt -S -globalopt < %s | FileCheck %s --check-prefixes=ALL,SECOND
;
; Make sure we use FIRST to check for @f as ALL does not work.

define internal void @f() {
  ret void
}
