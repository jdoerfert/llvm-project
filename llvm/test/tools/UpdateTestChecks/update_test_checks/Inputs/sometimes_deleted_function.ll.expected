; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
; RUN: opt -S < %s | FileCheck %s --check-prefixes=ALL,FIRST
; RUN: opt -S -globalopt < %s | FileCheck %s --check-prefixes=ALL,SECOND
;
; Make sure we use FIRST to check for @f as ALL does not work.

define internal void @f() {
; FIRST-LABEL: @f(
; FIRST-NEXT:    ret void
;
  ret void
}
